update doc to reflect the custom ops timeout (#2037)
thin-edge/thin-edge.io: docs/src/tutorials/supported_operations.md

In this example we want `thin-edge.io` to pick up a message on specific topic and execute the command on the device, our topic is `c8y/s/ds`.
We also know that the message we expect is going to use SmartRest template `511` and our plugin is located in `/etc/tedge/operations/command`.

> The operation is configured to `timeout` after 10 seconds.

Author: done

> ```admonish note

Author: done

> To avoid a custom operation from being run for longer, the custom operation can be configured to `timeout` after some time.

Reviewer: I'd drop this line if the above suggestion is accepted.

Author: done

> The `timeout` that is configured will be in seconds. If a custom operation is not configured with a `timeout` value,
> then it will use default `timeout`,.i.e. 3600 seconds.

Reviewer: ```suggestion
The `timeout` that is configured will be in seconds.
If a custom operation is not configured with a `timeout` value, then it will use default the `timeout` of 3600 seconds.
```

It's a good markdown practice to keep individual sentences on separate lines to make them review-friendly.

Author: Ok, thanks. done.

> If the operation does not complete within that `timeout` period, then the operation will be stopped.

Author: done
